pinot-controller/src/main/java/org/apache/pinot/controller/api/access/AccessControlFactory.java

```diff
@@ -30,5 +32,8 @@ default void init(PinotConfiguration pinotConfiguration) {
     // left blank
   }
 
+  default void init(ZkHelixPropertyStore<ZNRecord> zkHelixPropertyStore) {
+  }
+
```

Comment:
similar to broker, backwards-compatibility is key. I suggest a `init(ControllerConf, ZkHelixPropertyStore<ZNRecord>)` here that calls `init(ControllerConf)` by default

Reply:
@apucher Thanks for your suggestions. I introduce an init(PinotConfiguration, ZkHelixPropertyStore) that defaults to invoking the legacy init(PinotConfiguration) method Rather that using instanceof. Looking forward your review again.